controllers/util.go

```diff
@@ -177,3 +186,29 @@ func equals(permissionTypeA stardogrest.Permission, permissionTypeB StardogPermi
 
 	return action && resourceType && resources
 }
+
+var apiClientMap = sync.Map{}
```

Comment:
Do we really need the complexities of having a client cache? What happens for example of the credentials change without restarting the controller?

Reply:
Removed the client cache 👍